### PR TITLE
update version of smarty thru composer to fix imdbonline issue

### PR DESCRIPTION
**Pull request by copperhead57**
imdbonline requires updated version of smarty. implementing thru composer

imdbonline search requires reinstatement of quick fix due to more javascript changes

### CODE REVIEW

**Comment by andig** [comment]
Fix #125

File: composer.json [hunk 1]
```diff
@@ -2,7 +2,8 @@
     "name": "andig/videodb",
     "type": "project",
     "require": {
-        "guzzlehttp/guzzle": "^6.3"
+        "guzzlehttp/guzzle": "^6.3",
+        "smarty/smarty": "v3.1.34"
```
Comment (andig) [inline composer.json]: Do you need the specific version or rather ^3.1?

Comment (copperhead57) [reply composer.json]: have only tested with 3.1.34 as i believe that is the latest stable version

Comment (copperhead57) [reply composer.json]: don't need specific version ^3.1 should be ok.
my ide tool didnt allow for non specific entry.
apologies for confusion.

Comment (andig) [reply composer.json]: Please update PR ;)

Comment (copperhead57) [reply composer.json]: composer.json updated and committed

**Pull request merged**